webpack.config.js

```diff
@@ -93,120 +93,163 @@ module.exports = (env, argv) => {
         },
   };
 
-  return {
```

Comment:
You're changing around a *lot* of stuff in the webpack.config now. How much of this is needed for configuring the worker, vs how much is general reformating/refactoring?

Reply:
Most of it is whitespace change, most of the rest of it is refactoring to avoid  duplicating code. 

This is using https://webpack.js.org/configuration/configuration-types/#exporting-multiple-configurations to have a second configuration for the custom worker file, which needs to specify some of the same options as the main configuration.